src/connector/src/sink/clickhouse.rs

```diff
@@ -534,6 +600,15 @@ impl AsyncTruncateSinkWriter for ClickHouseSinkWriter {
     ) -> Result<()> {
         self.write(chunk).await
     }
+
+    async fn barrier(&mut self, is_checkpoint: bool) -> Result<()> {
+        if let Some(inserter) = self.inserter.take()
```

Comment:
Should we expose a config for user to determine the batch size of each insert?

Reply:
It doesn't seem to work, the ck rust interface doesn't allow it to be configured

Reply:
I mean does it make sense to end (flush) an inserted based on batch size?